### PR TITLE
update scripts

### PR DESCRIPTION
**Pull request by i5possible**

### CODE REVIEW

File: packages/scan/auto/prod-start [hunk 1]
```diff
@@ -0,0 +1,8 @@
+docker run -d --rm \
+    --name parallel-scan \
+    -e VERSION=0.1.0 \
```
Comment (crystalynliu) [inline packages/scan/auto/prod-start:3]: What's the version for?

File: packages/scan/auto/prod-start [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    -e PARALLEL_ENDPOINT=wss://testnet-relay-rpc.parallel.fi #FIX ME\
+    -e MONGO_URL=mongodb://192.168.0.103:27017 #FIX ME\
+    -e BLOCK_NUMBER=0 #FIX ME\
+    -e BUSINESS=referral \
+    parallel-scan:latest
```
Comment (crystalynliu) [inline packages/scan/auto/prod-start:8]: should we run the `yarn make-image` before running this script? Could you write down the step of deployment somewhere?

File: packages/scan/auto/prod-start (hunk 1 above)
Comment (i5possible) [reply packages/scan/auto/prod-start]: removed

File: packages/scan/auto/prod-start (hunk 2 above)
Comment (i5possible) [reply packages/scan/auto/prod-start:7]: Yes. We need to build and push the image to dockerhub first. It should be done in the CI.

**Pull request merged**